### PR TITLE
(SP: 1) Write unit test for "Subjects" page #101

### LINKED ISSUE #101: (SP: 1) Write unit test for "Subjects" page
### Page unit test
Unit test for "**Subjects**" page
Scenaries descriptions:
- [ ] It should render page
- [ ] Text of the page must be in the document

[Link to component](https://github.com/Space2Study-UA-1156/Client-02/blob/develop/src/pages/subjects/Subjects.jsx)

Current coverage:
Image

### PR DESCRIPTION
**Pull request by Nazar-Lysak**
- It should render page
-  Text of the page must be in the document

image

### CODE REVIEW

**Review by m6xcom — APPROVED** [review]

**Comment by docia** [comment]
- It should render page - done by your test. 
- Text of the page must be in the document - NOT done. I'd advise you to use 'getByText' and 'toBeInTheDocument' to check for the document's text.  
+ based on Vladyslav's comment on my task, it is also advisable to mock the PageWrapper component. 

@vlad-khrychov please give your opinion on my comment.

**Review by docia — COMMENTED** [review]
- It should render page - done by your test.
- Text of the page must be in the document - NOT done. I'd advise you to use 'getByText' and 'toBeInTheDocument' to check for the document's text.

+ based on Vlad's comment on my task, it is also advisable to mock the PageWrapper component.

@vlad-khrychov please give your opinion on my comment.

**Comment by Nazar-Lysak** [comment]
Hi @docia ,
Thank you for your feedback. I agree and will update the test as you suggested

**Review by docia — APPROVED** [review]

**Review by Olenka-Hryk — APPROVED** [review]

**Comment by vlad-khrychov** [comment]
Yes, if `PageWraper` mocked, you can use `test-id` in this mock. So, in the first test `getByTestId` will be used  and `getByText` will be used in second `it`.

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]
Mock `PageWraper` and get mocked div by test id in the first test,

File: tests/unit/pages/subjects/Subjects.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { renderWithProviders } from '~tests/test-utils'
+import { render, screen } from '@testing-library/react'
```
Comment (vlad-khrychov) [inline tests/unit/pages/subjects/Subjects.spec.jsx]:
```suggestion
import { render, screen } from '@testing-library/react'
import { renderWithProviders } from '~tests/test-utils'
```

**Review by vlad-khrychov — APPROVED** [review]
Now it looks great!

**Pull request merged**